testutils/infra/container_manager/base.py

```diff
@@ -0,0 +1,14 @@
+
+class BaseContainerManagerNamespace:
+    """Base class to define a containers namespace
+    """
+
+    def setup(self):
```

Comment:
I suppose at least setup shall be implemented by child classes, I would `raise NotImplementedError`